guides/micronaut-openapi-generator-server/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "asciidoctor": "micronaut-openapi-generator-server.adoc",
+  "slug": "micronaut-openapi-generator-server",
+  "title": "Use OpenAPI Definition to Generate a Micronaut Server",
+  "intro": "Learn how to write an OpenAPI definition, use it to generate a server template for Micronaut framework, and get it all to work",
+  "authors": ["Andriy Dmytruk"],
+  "tags": ["openapi", "server"],
+  "category": "Getting Started",
+  "languages": ["java"],
+  "buildTools": ["maven", "gradle"],
```

Comment:
delete this line, no need to specify `buildTools` if you're using both Gradle and Maven